### DISCUSSION TITLE
ESP32-POE statische LAN IP?

### DISCUSSION THREAD

**Discussion by x90cr** [tbnobody/OpenDTU#685] [General]
Kann ich am ESP32-POE irgendwie eine statische LAN IP festlegen?

**Comment by huckes** [comment]
Eine Statische IP kannst Du doch am Router festlegen anhand der MAC-Adresse.

**Reply by x90cr** [reply]
Das dass überhaupt geht wusste ich noch gar nicht. Die Fritzbox unterstütz das tatsächlich.
Ist aber keine besonders gute und saubere Lösung.
Ich muss morgen mal schauen ob ich die damit auch in den statischen Adressbereich zwingen kann.

**Reply by huckes** [reply]
Warum willst Du überhaupt eine statische IP zuweisen? In welcher Anwendungsumgebung ist das "sauber"?
Die Fritz-Box behält eine einmal zu einer MAC-Adresse vergebenen IP-Adresse üblicherweise bei und würfelt nicht täglich eine neue. 
Statische IPs am Gerät einstellen kenne ich nur aus dem IT-Mittelalter mit 10BaseT. Da hatten wir sowas in der Firma. Und unsere IT-Leute haben sich die Turschuhe abgelaufen um doppelt vergebene Adressen aufzuspüren.

**Reply by x90cr** [reply]
Die IP's im Netzwerk sind zum großen Teil in NodeRed verwurstet und da müssen die 100% zuverlässig funktionieren.
Wenn mir ein Gerät ausfällt dann wird das gegen ein neues ersetzt und nur einmal die IP wieder gesetzt.
Wenn mir die Fritzbox ausfällt wird das so schon lustig die Portweiterleitungen wieder einzurichten, da brauche ich nicht noch unzählige Geräte die damit ihre IP's verlieren. Es geht dabei auch um Ausfallsicherheit. Wenn mir die Fritzbox stirbt habe ich kein Internet ok, aber Hausintern kommuniziert alles weiter.

**Reply by huckes** [reply]
Ich denke, mit der aktuellen Version wird das nicht möglich sein. Da müsstest Du mal selbst den Code anpassen. Hinweise, wie Du da ne statische IP reinbringst, findest Du im Netz, u.a. auch im Olimex-Support Forum.
In der Fritzbox solltest Du auch den Adressbereich aus der DHCP-Adressvergabe ausklammern, den Du mit statischen IPs belegst.

**Reply by huckes** [reply]
Uups, siehe Antwort von tbnobody weiter unten: DHCP im Web-UI deaktivieren und dann einfach eintragen. Sorry, hatte ich nicht gesehen.

**Comment by tbnobody** [comment]
Hast du versucht die statische IP einfach im Webinterface zu setzen? Dort wo man auch die IP für das WiFi Interface setzen kann? Es gibt keine separate Config, aber das was man dort einträgt wird im falle eines aktiven Ethernet Interfaces auch dort angewendet.
